src/test/java/com/wikia/webdriver/elements/common/CommunityHeader.java

```diff
@@ -62,6 +64,21 @@
   @FindBy(css = ".wds-community-header")
   private WebElement communityHeader;
 
+  @FindBy(css = ".wds-community-header__wiki-buttons .wds-dropdown")
+  private WebElement moreToolsDropdown;
+
+  @FindBy(css = ".wds-community-header__wiki-buttons a[data-tracking=\"more-add-new-image\"]")
```

Comment:
I would be cautious about using data-tracking for testing purposes, are there any other selectors that we can use?

Reply:
@hakubo currently it is the only way to distinguish those links apart from the order. If we decide to remove these tracking label then we can add some specific selectors, however I don't think it will be soon